tools/clang/test/HLSLFileCheck/d3dreflect/rdat_mintarget/sm68_samplecmpgradbias.hlsl

```diff
@@ -60,7 +60,7 @@ void samplecmpbias() {
 // RDAT:   MinShaderTarget: 0x50068
 
 [shader("compute")]
-[numthreads(1,1,1)]
+[numthreads(4,1,1)]
```

Comment:
I see why this is required. I'm not sure what about this change enforced that requirement.

Reply:
Ah, the change that enforces this requirement was separated into another branch for independent PR, but forgot to move this change I guess.  Should I remove it or leave it (it is correct).